models/migrations/migrations.go

```diff
@@ -234,6 +234,8 @@ var migrations = []Migration{
 	NewMigration("add commit status context field to commit_status", addCommitStatusContext),
 	// v89 -> v90
 	NewMigration("add original author/url migration info to issues, comments, and repo ", addOriginalMigrationInfo),
+	// v90 -> v91
+	NewMigration("change length of some repository columns", changeSomeColumnsLengthOfRepo),
```

Comment:
If backport to 1.9 is ok for new migration?

Reply:
@techknowlogick No. When back port to 1.9, I will remove this migration and it should work.

Reply:
Ok. Thank you :)